scenarios/pravega-flink-connector-sql-samples/src/main/java/io/pravega/connectors/nytaxi/PrepareMain.java

```diff
@@ -48,15 +42,13 @@ public void handleRequest() {
             createStream();
         }
 
-        Stream streamInfo = getPravegaConfig().resolve(Stream.of(getScope(), getStream()).getScopedName());
+        Stream streamInfo = Stream.of(getScope(), getStream());
 
-        TypeInformation<Row> typeInfo = (RowTypeInfo) TypeConversions.fromDataTypeToLegacyInfo(TripRecord.getTableSchema().toRowDataType());
-
-        FlinkPravegaWriter<Row> writer = FlinkPravegaWriter.<Row>builder()
+        FlinkPravegaWriter<TripRecord> writer = FlinkPravegaWriter.<TripRecord>builder()
                 .withPravegaConfig(getPravegaConfig())
                 .forStream(streamInfo)
-                .withSerializationSchema(new JsonRowSerializationSchema.Builder(typeInfo).build())
-                .withEventRouter(event -> String.valueOf(event.getField(6)))
+                .withSerializationSchema(new PravegaSerializationSchema<>(new JsonSerializer<>(TripRecord.class)))
+                .withEventRouter(r -> String.valueOf(r.getRideId()))
```

Comment:
How do I make sure that field 6 previous was actually the ride id? I'm assuming we are not trying to change the behavior of the sample, so I'd like to make sure that we aren't changing it.

Reply:
Actaully I changed it here, it was the